test/index.js

```diff
@@ -240,14 +320,22 @@ describe('normalize.undefined', function() {
     expect(result).toEqual(value);
     done();
   });
+
+  it('rejects values that won\'t coerce to undefined', function(done) {
```

Comment:
Ah,  I missed the collapsed button.  I assumed it was supposed to be added to the `function` tests because you didn't add a "rejects" test to that block like you did for every other one.